VTUL/dlp-access
# LIBTD-2386: Modify cypress tests to use Demo site as test data

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2386) (:star:)

# What does this Pull Request do? (:star:)
This PR modifies cypress integration tests so that these end-to-end tests are running against the data hosted in demo site.

# What's the changes? (:star:)

* Changes tests according to the data in demo site such as search results and displayed metadata
* Changes tests according to the admin configurations in demo site
* admin/HomePageForm.js: replaced the hardcoded site info with environment variable

# How should this be tested?

* Run the tests with "REACT_APP_REP_TYPE=Default" and check if all tests are passed

# Interested parties
Tag (@yinlinchen ) interested parties

(:star:) Required fields
